### PR TITLE
PoC: Support swagger ``example`` field for colander

### PR DESCRIPTION
**Pull request by elbart**
This tackles one part of #85.

### CODE REVIEW

File: cornice_swagger/converters/schema.py [hunk 1]
```diff
@@ -126,6 +126,8 @@ def convert_type(self, schema_node):
             converted['description'] = schema_node.description
         if schema_node.default is not colander.null:
             converted['default'] = schema_node.default
+        if 'example' in schema_node.__dict__:
```
Comment (gabisurita) [inline cornice_swagger/converters/schema.py:129]: If checking for `schema_node.example` doesn't work, I would rather use:
```python
example = getattr(schema_node, 'example', None)
if example is not None:
    ...
```

**Review by gabisurita — APPROVED** [review]
Thanks for the contribution and sorry for taking too long to review this. 

I'm 👍 for merging this. I'm just missing mentioning it on the documentation.

**Comment by elbart** [comment]
@gabisurita thanks for the feedback. I will fix your comments, but will also try to add more stuff about examples. The example field I added here, was more for the request body. For response definitions, the examples seem to be different. Will follow up on this. Please do not merge for now!

Thanks!

**Comment by fmigneault** [comment]
@gabisurita 
Any plans on merging this functionality soon? My schemas are already defined with the `example` kw but just need cornice-swagger to added them to the api docs generation process.

**Comment by gabisurita** [comment]
@fmigneault I'll probably merge this as is in the next release. Sorry for the delay.

**Comment by gabisurita** [comment]
Released in 0.7.0

https://github.com/Cornices/cornice.ext.swagger/releases/tag/0.7.0

**Pull request merged**